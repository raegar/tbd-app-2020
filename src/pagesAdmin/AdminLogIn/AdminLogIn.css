.admin-login-container{
    width: 100vw !important;

    display: flex;
    margin: 0;
    padding: 0;
}