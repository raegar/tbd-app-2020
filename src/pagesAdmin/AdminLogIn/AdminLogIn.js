import React, {useState} from 'react';
import {Row, Col} from 'react-bootstrap';
import './AdminLogIn.css';
import TextBox from '../../components/TextBox/TextBox';
import AdminNavBar from '../../componentsAdmin/AdminNavBar/AdminNavBar';

function AdminLogIn() {
	//const [errorMessage, setErrorMessage] = useState("");
	//const [emailInput, setEmailInput] = useState("");
	//const [passwordInput, setPasswordInput] = useState(null);
	/*
	function loginButtonPressed(e) {
		//for now we just check to see if there is a password
		let validated = !!passwordInput;
		setErrorMessage(emailInput);

		if (!validated) {
			e.preventDefault();
		}
	}*/

	return (
		<div className="admin-login-container">{/*LogIn*/}
			<Col md={2.5} className="first-col">
				<AdminNavBar theActive="LogIn"/>
				{/*<UCPButton
						to="/"
						buttonText="Log In"
						className="mediumbutton"
						onClick={loginButtonPressed}
					/>*/}
			</Col>
			<Col className="second-col">
				<Row>
					<p>{errorMessage}</p>
				</Row>
				<Row>
					<TextBox 
						title="Email" 
						placeholder="Email" 
						//onChange={(e) => setEmailInput(e.target.value)}
					/>
				</Row>
				<Row>
					<TextBox 
						title="Password" 
						placeholder="Password" 
						//onChange={(e) => setPasswordInput(e.target.value)}
						type="password"
					/>
				</Row>
			</Col>
		</div>
	);
}

export default AdminLogIn;