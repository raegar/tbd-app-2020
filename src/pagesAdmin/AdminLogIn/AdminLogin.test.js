import React from 'react';
import { render, } from '@testing-library/react';
import '@testing-library/jest-dom';
import { BrowserRouter as Router} from 'react-router-dom';
import AdminLogIn from './AdminLogIn';

it("Renders AdminLogIn", () => {
	const { asFragment } = render(<Router><AdminLogIn /></Router>);
	expect(asFragment()).toMatchSnapshot();
});
test ("AdminLogin Conditional rendering"), ()=> {
	const lookingFor ='Email:'
	render(
		<Router>
			<AdminLogIn />
		</Router>
	);
});