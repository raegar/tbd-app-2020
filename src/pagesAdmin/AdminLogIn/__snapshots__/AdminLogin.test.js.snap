// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders AdminLogIn 1`] = `
<DocumentFragment>
  <div
    class="admin-login-container"
  >
    <div
      class="first-col col-md-2.5"
    >
      <div
        class="navbar-container"
      >
        <a
          href="/AdminDashboard"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminDashboard"
            type="button"
          >
            Dashboard​
          </button>
        </a>
        <a
          href="/AdminData"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminData"
            type="button"
          >
            Data​
          </button>
        </a>
        <a
          href="/AdminGraph"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminGraph"
            type="button"
          >
            Gender​
          </button>
        </a>
        <a
          href="/AdminGraph"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminGraph"
            type="button"
          >
            Age​
          </button>
        </a>
        <a
          href="/AdminGraph"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminGraph"
            type="button"
          >
            Course​
          </button>
        </a>
        <a
          href="/AdminGraph"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminGraph"
            type="button"
          >
            Ethnicity​
          </button>
        </a>
        <a
          href="/AdminGraph"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminGraph"
            type="button"
          >
            Location​
          </button>
        </a>
        <a
          href="/AdminGraph"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminGraph"
            type="button"
          >
            Income​
          </button>
        </a>
        <a
          href="/AdminGraph"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminGraph"
            type="button"
          >
            Disabilities​
          </button>
        </a>
        <a
          href="/AdminGraph"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminGraph"
            type="button"
          >
            Support​
          </button>
        </a>
        <a
          href="/AdminGraph"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminGraph"
            type="button"
          >
            Education​
          </button>
        </a>
        <a
          href="/AdminGraph"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminGraph"
            type="button"
          >
            Employment​
          </button>
        </a>
        <a
          href="/AdminGraph"
        >
          <button
            class="admin-navbar-button"
            data-testid="ucpButton"
            href="/AdminGraph"
            type="button"
          >
            Payment​
          </button>
        </a>
        <a
          href="/AdminLogIn"
        >
          <button
            class="admin-navbar-button-active"
            data-testid="ucpButton"
            href="/AdminLogIn"
            type="button"
          >
            LogIn​
          </button>
        </a>
      </div>
    </div>
    <div
      class="second-col col"
    >
      <div
        class="row"
      >
        <p />
      </div>
      <div
        class="row"
      >
        <div
          class="mobiletextbox"
        >
          <label>
            Email
          </label>
          <input
            maxlength="30"
            minlength="0"
            placeholder="Email"
            size="20"
            type="text"
          />
        </div>
      </div>
      <div
        class="row"
      >
        <div
          class="mobiletextbox"
        >
          <label>
            Password
          </label>
          <input
            maxlength="30"
            minlength="0"
            placeholder="Password"
            size="20"
            type="password"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
